src/main/java/duke/command/Duke.java

```diff
@@ -0,0 +1,231 @@
+                    taskList.add(new Event(taskName, taskTime));
+                    String response = LINE_SEPARATOR
+                            + SPACING + "added: " + taskList.get(taskList.size() - 1) + "\n"
+                            + LINE_SEPARATOR;
+                    System.out.println(response);
+                    writeData();
+                } catch (StringIndexOutOfBoundsException e) {
+                    String error = LINE_SEPARATOR
+                            + SPACING + NO_ARGUMENT_1 + "event" + NO_ARGUMENT_2 + "\n"
+                            + LINE_SEPARATOR;
+                    System.out.println(error);
+                }
+
+                break;
+            case "todo":
+                try {
+                    String taskName = line.substring("todo ".length());
+                    taskList.add(new ToDo(taskName));
+                    String response = LINE_SEPARATOR
+                            + SPACING + "added: " + taskList.get(taskList.size() - 1) + "\n"
+                            + LINE_SEPARATOR;
+                    System.out.println(response);
+                    writeData();
+                } catch (StringIndexOutOfBoundsException e) {
+                    String error = LINE_SEPARATOR
+                            + SPACING + NO_ARGUMENT_1 + "todo" + NO_ARGUMENT_2 + "\n"
+                            + LINE_SEPARATOR;
+                    System.out.println(error);
+                }
+                break;
+            case "delete":
+                try {
+                    int currentTask = Integer.parseInt(line.split(" ")[1]) - 1;
+                    String update = LINE_SEPARATOR
+                            + SPACING + "Noted! I've removed this task: \n"
+                            + SPACING + taskList.get(currentTask) + "\n"
+                            + LINE_SEPARATOR;
+                    System.out.println(update);
+                    taskList.remove(currentTask);
+                    writeData();
+                } catch (IndexOutOfBoundsException e) {
+                    String error = LINE_SEPARATOR
+                            + SPACING + NO_ARGUMENT_1 + "delete" + NO_ARGUMENT_2 + "\n"
+                            + LINE_SEPARATOR;
+                    System.out.println(error);
+                }
+                break;
+            default:
+                String response = LINE_SEPARATOR
+                        + SPACING + CANNOT_IDENTIFY + "\n"
+                        + LINE_SEPARATOR;
+                System.out.println(response);
+            }
+        }
+
+        String Bye = LINE_SEPARATOR
+                + SPACING + "Bye. Hope to see you again soon!\n"
+                + LINE_SEPARATOR;
+        System.out.println(Bye);
+    }
+
+
+    public static void list() {
+        String listOutput = LINE_SEPARATOR;
+        for (int i = 0; i < taskList.size(); i++) {
+            listOutput += SPACING + (i + 1) + ". "
```

Comment:
This expression is slightly complicated. I suggest extracting it further for improved clarity.